awkward-cpp/README.md

```diff
@@ -0,0 +1,3 @@
+# `awkward-cpp`
+
+`awkward-cpp` provides the `awkward` package with a suite of high-performance (compiled) kernels which facilitate computations on ragged arrays. It also implements performance-sensitive features, such as the `ak.ArrayBuilder()` class.
```

Comment:
The list of features was missing AwkwardForth and the header-only classes. Instead of making a list that could get out of date, let's just say that they probably don't want this on its own.

```suggestion
`awkward-cpp` provides precompiled routines for the `awkward` package. It is not useful on its own, only as a dependency for `awkward `.
```